### ISSUE TITLE
ProgressIndicator: color contrast not met in dark mode with information value state [a11y issue]

### ISSUE THREAD

**Issue by tkeiner** [SAP/ui5-webcomponents#11031]
### Bug Description

When the ProgressIndicator Component is used in the "information" value state mode and the dark horizon theme is used, then the color contrast of the progress bar does not meed the minimal contrast ratio of 4.5 : 1 as defined in the WCAG guidelines.

Example: <ui5-progress-indicator value="40" value-state="Information"></ui5-progress-indicator>

![Image](https://github.com/user-attachments/assets/99844e3a-3d90-4fff-9276-8ab6721d8022)

I did not check the contrast for the other value state options.

### Affected Component

ProgressIndicator

### Expected Behaviour

Color contrast should be minimum of 4.5 : 1

### Priority

None

### UI5 Web Components Version

2.5.1

### Browser

Chrome

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by PetyaMarkovaBogdanova** [comment]
Hi @tkeiner ,
the styles of the ui5-progress-bar are implemented by design spec. You should follow the innerSource process to log an issue to the design team.
best regards,
Petya Markova.
UI5 Web Components Dispatcher
